assets/js/base.js

```diff
@@ -0,0 +1,8 @@
+$(window).scroll(function() {
+// 100 = The point you would like to fade the nav in.
+  if ($(window).scrollTop() > 0 ){
+    $('html').addClass('trans');
+  } else {
+    $('html').removeClass('trans');
+  };
+});
```

Comment:
this is causing interesting effects when there isn't enough content on the page. the logo resizes